host_cleanup.sh

```diff
@@ -42,14 +42,11 @@ ANSIBLE_FORCE_COLOR=true ansible-playbook \
     -i vm-setup/inventory.ini \
     -b -v vm-setup/teardown-playbook.yml
 
-if [ "$USE_FIREWALLD" == "False" ]; then
- ANSIBLE_FORCE_COLOR=true ansible-playbook \
-    -e "{use_firewalld: $USE_FIREWALLD}" \
-    -e "external_subnet_v4: ${EXTERNAL_SUBNET_V4}" \
+ANSIBLE_FORCE_COLOR=true ansible-playbook \
```

Comment:
I am afraid if we remove the condition it will re enable the firewalld instead of cleaning them not yet sure if we can use disabled with firewalld in the cleaning step

Reply:
My question in the other PR if I enabled something was enabled by default does make sense to disable it in the cleaning step ?

Reply:
if `use_firewalld` is true then the firewall rules would not be cleaned up correctly if it is nod true it doesn't matter anyways because then the iptable rules will be used.